app/javascript/react/src/containers/ReviewFormContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, {Component} from 'react';
+
+class ReviewFormContainer extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      reviewBody: '',
+      rating: 0
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.clearForm = this.clearForm.bind(this)
+  }
+
+  handleChange(event){
+    let value = event.target.value
+    let name = event.target.name
+    this.setState({ [name]: value})
+  }
+
+  clearForm(){
+    this.setState({
+      reviewBody: '',
+      rating: 0
+    })
+  }
+
+  handleSubmit(event){
+    event.preventDefault();
+    if(this.state.reviewBody == '' || this.state.rating == 0){
+
```

Comment:
You don't have anything happening in this part of the conditional. Please clean this up!